### PR TITLE
Adding dYdX to Explore page

### PR DESCRIPTION
**Pull request by chriskalani**
Screen Shot 2019-06-26 at 5 11 20 PM

### CODE REVIEW

File: packages/website/ts/types.ts [hunk 1]
```diff
@@ -255,6 +255,7 @@ export interface ExploreProject {
     url: string;
     keywords: string[];
     instant?: ExploreProjectInstantMetadata;
+    logo_max_width?: string;
```
Comment (fragosti) [inline packages/website/ts/types.ts]: camelCase -> `logoMaxWidth`

Comment (fragosti) [reply packages/website/ts/types.ts]: Oh realized we're doing this snake case stuff elsewhere... shouldn't be that way :p

**Comment by AntonioJuliano** [comment]
😄

File: packages/website/ts/types.ts (hunk 1 above)
Comment (dave4506) [reply packages/website/ts/types.ts]: I think the snake case is on me, I was confused it was snake case as well; dunno what the heck I was thinking when I wrote it.

**Pull request merged**